docs/section15/Rule15-3.md

```diff
@@ -0,0 +1,20 @@
+# Transformers - Rule 15-3
+**Rule ID:** 15-3  
+**Rule Description:** User RMR transformer Name in Proposed RMR  
+**Rule Assertion:** Proposed RMR = expected value  
+**Appendix G Section:** Transformers  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None  
+**Evaluation Context:**  Each Data Element   
+**Applicability Checks:**  
+ 1. Number of transformers in User RMR greater than 0  
+ 
+**Manual Checks:** None  
+
+## Rule Logic:
+- **Applicability Check 1:** `length(U_RMR.transformers) > 0:`  
```

Comment:
It occurs to me that for list-type rules such as this, instead of using an applicability check for a non-empty list, we could instead give the result: [] for an empty list. This would actually provide more precise information that the more general result: 'NA'.  If we decide to handle empty lists this way, we do not need to include Applicability Checks like line 15 everywhere.